.rubocop.yml

```diff
@@ -1,2 +1,2 @@
 inherit_from:
-  - https://raw.githubusercontent.com/theforeman/foreman/4aa0bb6a24b45347ff9b673055499763eec72489/.rubocop.yml
+  - https://raw.githubusercontent.com/theforeman/foreman/develop/.rubocop.yml
```

Comment:
This will cause rubocop to fail each time core changes its definition.

Reply:
IMO each time it will be a very small change,
instead of updating to the latest commit with lots of changes to our files.